test/UnitTest/Microsoft.AspNet.OData.Test.Shared/OpenEntityTypeTest.cs

```diff
@@ -333,11 +333,13 @@ public async Task Post_OpenEntityType()
             const string Payload = "{" +
               "\"@odata.context\":\"http://localhost/odata/$metadata#OpenCustomers/$entity\"," +
               "\"CustomerId\":6,\"Name\":\"FirstName 6\"," +
-              "\"Address\":{" +
+              "\"Address\":{" +                
                 "\"Street\":\"Street 6\",\"City\":\"City 6\",\"Place\":\"Earth\",\"Token@odata.type\":\"#Guid\"," +
                 "\"Token\":\"4DB52263-4382-4BCB-A63E-3129C1B5FA0D\"," +
-                "\"Number\":990" +
+                 "\"@NS1.abc\": \"Test\"," +
+                "\"Number\":990" +                
               "}," +
+              "\"Website@NS.Street\": \"Test\"," +
```

Comment:
>"\"Website@NS.Street\": \"Test\"," + [](start = 14, length = 36)

where is the verification about these?
We should make sure the action parameter has the instance annotation added?